client/src/components/pages/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classes from './../../styles/pages/Login.module.css'
+
+const Logging = () => {
```

Comment:
1) Лучше чтобы компонент назывался как и модуль
2) Лучше в проекте использовать везде один и тот же способ объявления компонент: обычные функции вместо стрелочных (но это на самом деле вообще некритично, просто имеем в виду, что код лучше везде писать по одинаковому)
```suggestion
export default function Login() {
```